gateway-ha/pom.xml

```diff
@@ -81,30 +81,53 @@
             <groupId>io.dropwizard</groupId>
             <artifactId>dropwizard-assets</artifactId>
         </dependency>
-
         <dependency>
             <groupId>io.dropwizard</groupId>
             <artifactId>dropwizard-auth</artifactId>
         </dependency>
-
+        <dependency>
```

Comment:
Please add empty lines.

Reply:
Is this the trino style even for dependencies that belong together? Its soooo verbose

Reply:
That's the style in Trino repository.

Reply:
ok.. will update